packages/ds-sam-sdk/src/auction.ts

```diff
@@ -375,6 +393,8 @@ export class Auction {
           }
         }
       }
+      // if totalScalable = 0, we'll get NaN which is caught below resulting in
```

Comment:
```suggestion
      // if totalScalable = 0, we'll get Infinity or NaN which is caught below resulting in
```
(minor note - if `leftToScale` is above `0` then we get `Infinity` instead of `NaN`)